.github/workflows/pylint.yml

```diff
@@ -14,7 +14,9 @@ on:
 
 jobs:
   pylint:
-    runs-on: ubuntu-latest
+    # Need to specify 20.04, because ubuntu-latest does not work with
+    # python 3.6: https://github.com/actions/setup-python/issues/355#issuecomment-1335042510
+    runs-on: ubuntu-20.04
```

Comment:
(Curious: why have we been passing these tests?)

Reply:
The GitHub runner recently updated the OS used by `ubuntu-latest` to `ubuntu-22.04`, which does not support python 3.6 anymore, as mentioned  in https://github.com/actions/runner-images/issues/6399.
We may also want to consider deprecating the Python 3.6 support #609 
https://github.blog/changelog/2022-12-01-github-actions-larger-runners-using-ubuntu-latest-label-will-now-use-ubuntu-22-04/